api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9280,5 +9280,23 @@
 			<column name="sort_weight" type="double"></column>
 		</addColumn>
 	</changeSet>
+
+
+	<changeSet id="201604030003" author="vishnuraom">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">SELECT count(*) FROM drug_order WHERE dosing_type = 'SIMPLE' OR dosing_type = 'FREE_TEXT';</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Converting values in drug_order.dosing_type column(TRUNK-4845)</comment>
+		<update tableName="drug_order">
+			<column name="dosing_type" value="org.openmrs.SimpleDosingInstructions"/>
+			<where>dosing_type = 'SIMPLE'</where>
+		</update>
+		<update tableName="drug_order">
+			<column name="dosing_type" value="org.openmrs.FreeTextDosingInstructions"/>
```

Comment:
I've read https://issues.openmrs.org/browse/TRUNK-4845 to understand why the classname is stored here. It's kind of weird. What happens if the class is refactored? Is there any good reason to do this ?